content/geometry/ManhattanMST.h

```diff
@@ -0,0 +1,36 @@
+/**
+ * Author: chilli, Takanori MAEHARA
+ * Date: 2019-11-02
+ * License: CC0
+ * Source: https://github.com/spaghetti-source/algorithm/blob/master/geometry/rectilinear_mst.cc
+ * Description: Returns up to 8*N edges, which are guaranteed to contain the
```

Comment:
Add that it's for the 2d case with integer coordinates, and maybe explain that Manhattan MST means.

Reply:
Actually, I think I'm gonna modify it to work with ints/doubles - I think it's fairly reasonable to call this on doubles.

Reply:
Actually, nvm, that's pretty annoying...

Reply:
Ugh, doubles, now I need to sit down and analyze this for precision issues... I think it would be fine to use ints everywhere and leave it to users to change to doubles if needed; I don't think I've ever seen a problem which used Manhattan metric without integer coordinates.